Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2012-2014 Gabriel Gonzalez
```

Comment:
Thanks!  I'd like to hear what @CodaFi thinks, but I'd be very happy if did that.  :)